ibm_catalog.json

```diff
@@ -0,0 +1,460 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-cloud-logs",
+        "label": "Cloud automation for Cloud Logs",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "logging_monitoring"
+        ],
+        "keywords": [
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "cloud logs",
+          "observability",
+          "logging",
+          "monitoring",
+          "monitor",
+          "log"
+        ],
+        "short_description": "Creates and configures IBM Cloud Logs resources",
+        "long_description": "This architecture supports creating and configuring IBM Cloud Logs resources.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloud-logs/blob/main/README.md",
+        "offering_icon_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloud-logs/blob/main/images/observability-icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Cloud Logs",
+            "description": "Creates and configures a Cloud Logs instance."
+          },
+          {
+            "title": "Object Storage buckets for Cloud Logs",
+            "description": "Creates buckets required for the Cloud Logs instance."
+          }
+        ],
+        "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-cloud-logs/issues](https://github.com/terraform-ibm-modules/terraform-ibm-cloud-logs/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+        "flavors": [
+          {
+            "label": "Fully configurable",
+            "name": "fully-configurable",
+            "install_type": "fullstack",
+            "working_directory": "solutions/fully-configurable",
+            "configuration": [
+              {
+                "key": "ibmcloud_api_key"
+              },
+              {
+                "key": "prefix",
+                "required": true,
+                "description": "Prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+              },
+              {
+                "key": "region",
+                "required": true
+              },
+              {
+                "key": "existing_cos_instance_crn",
+                "required": true
+              },
+              {
+                "key": "existing_resource_group_name"
```

Comment:
This should be using the resource group picker. Please be consistent with SCC DA -> https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/7713daec73b967316e998bbd6faceb59de967947/ibm_catalog.json#L80-L88

This is include in the issue criteria too